lib/components/Shell/Shell.module.scss

```diff
@@ -41,7 +41,7 @@
 .nav-and-workspace {
     // lay out nav and workspace horizontally:
     display: flex;
-    height: 100%;
+    height: 100vh;

```

Comment:
How does this work? .shell already has a height:100vh, so height:100% here _should_ occupy the remaining available height, right?

Reply:
Disregard! @PatoBeltran found some unnecessary code that we could remove that produced the same results.